integration-test/src/main/java/com/sequenceiq/it/cloudbreak/util/ssh/action/SshEnaDriverCheckActions.java

```diff
@@ -0,0 +1,78 @@
+package com.sequenceiq.it.cloudbreak.util.ssh.action;
+
+import java.util.Collections;
+import java.util.stream.Stream;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.instancegroup.instancemetadata.InstanceMetaDataV4Response;
+import com.sequenceiq.cloudbreak.common.json.Json;
+import com.sequenceiq.cloudbreak.common.mappable.CloudPlatform;
+import com.sequenceiq.it.cloudbreak.CloudbreakClient;
+import com.sequenceiq.it.cloudbreak.dto.distrox.DistroXTestDto;
+import com.sequenceiq.it.cloudbreak.exception.TestFailException;
+import com.sequenceiq.it.cloudbreak.util.ssh.client.SshJClient;
+
+@Component
+public class SshEnaDriverCheckActions extends SshJClient {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(SshEnaDriverCheckActions.class);
+
+    private void checkEnaDriver(String instanceIP) {
+        String modinfoEnaCmd = "/usr/sbin/modinfo ena";
+        String wrongResult = "modinfo: ERROR: Module ena not found.";
+        Pair<Integer, String> result = executeCommand(instanceIP, modinfoEnaCmd);
+        if (result.getValue().startsWith(wrongResult)) {
+            throw new TestFailException("Cannot find ena driver on the instance");
+        }
+    }
+
+    private void checkEnaSupport(String instanceIP, String instanceId, String region) {
+        String modinfoEnaCmd = String.format("aws ec2 describe-instances --instance-ids %s --query \"Reservations[].Instances[].EnaSupport\" --region %s",
```

Comment:
I thought this will be implemented with an API call.

Reply:
I'm going to rewrite to an API call